### DISCUSSION TITLE
debian bookworm v12.7

### DISCUSSION THREAD

**Discussion by inindev** [inindev/debian-image#7] [Announcements]
debian bookworm 12.7

u-boot 2024.10

kernel versions:

rock-s0 (rk3508): inindev kernel 6.11.2-arm64

nanopi-r5c (rk3568): debian kernel 6.1.0-26-arm64

nanopi-r5s (rk3568): debian kernel 6.1.0-26-arm64

odroid-m1 (rk3568): debian kernel 6.1.0-26-arm64

radxa-e25 (rk3568): debian kernel 6.1.0-26-arm64

orangepi-5 (rk3588s): debian kernel 6.10.3-arm64

orangepi-5-plus (rk3588): debian kernel 6.10.11-arm64

nanopc-t6 (rk3588): debian kernel 6.10.11-arm64

rock-5b (rk3588): debian kernel 6.10.11-arm64

<hr /><em>This discussion was created from the release <a href='https://github.com/inindev/debian-image/releases/tag/v12.7'>debian bookworm v12.7</a>.</em>

**Comment by scott87** [comment]
Hi everyone. I have 2 nanopc t6 lts. I've been messing around for a while trying to set them as routers, homelab etc. But I was hoping to get access to kvm and iommu. I still can't find anything. Has the community gotten any further with pci pass through etc??? Thanks for all the work so far.
